Gateway/Command/CaptureCommand.php

```diff
@@ -53,8 +53,9 @@ public function execute(array $commandSubject)
         /** @var \Magento\Sales\Model\Order\Payment $payment */
         $payment = $commandSubject['payment']->getPayment();
 
-        $invoice = $this->registry->registry('current_invoice');
-        $payment->setSalesDocument($invoice);
+        if ($invoice = $this->registry->registry('current_invoice')) {
```

Comment:
Are you assigning the `$invoice` variable in the if statement? I think the code would be more readable if the assignment operation was done on the line before the if statement so that the variable assignment is not mistaken for an `==` operation.